xknx/config/schema.py

```diff
@@ -186,6 +200,10 @@ class LightSchema:
                     ),
                 }
             ),
+            vol.Optional(CONF_RED): COLOR_SCHEMA,
+            vol.Optional(CONF_GREEN): COLOR_SCHEMA,
+            vol.Optional(CONF_BLUE): COLOR_SCHEMA,
+            vol.Optional(CONF_WHITE): COLOR_SCHEMA,
```

Comment:
again I think this should share an inclusion group.

Reply:
fixed in https://github.com/XKNX/xknx/pull/508/commits/15cf578a43a8d07dcfa6f24a3f70ce410f1aaa55